### ISSUE TITLE
[BUG] Displaying uploaded image in canvas/kanban fails

### ISSUE THREAD

**Issue by diderich** [Leantime/leantime#1080]
**Describe the bug**
If I upload an image and add it to, for example, the `conclusion` of a canvas or the description of a `kanban`, the image is not displayed on screen.

**To Reproduce**
Note: *BASE_URL = https://www.didilan.net/dts/*

1. Go to the ideas menu and add an idea

2. Add an image to the idea
![step1](https://user-images.githubusercontent.com/10230144/198281008-7e7b84ba-254d-4613-8446-8a35907fb991.png)

3. Insert an image to an element: The image name includes the BASE_URL
![step2](https://user-images.githubusercontent.com/10230144/198280991-ab89cbe2-957a-44ea-8331-e56bbb9d6398.png)

4. Save the element
![step3](https://user-images.githubusercontent.com/10230144/198280948-35b1bf22-0dbb-4451-ad57-0327f921c349.png)

5. See that the image is not displayed
![step4](https://user-images.githubusercontent.com/10230144/198280880-36dfaf38-a3a2-4b0c-b4cd-de9dbdb3b18c.png)

6. Try editing the image - The system has stripped the BASE_URL part of the image URL
![step5](https://user-images.githubusercontent.com/10230144/198280937-97411f1b-5247-49ae-b237-0db50cf80b33.png)

*NOTE: The issue only happens with images uploaded on the local server. Images referenced through a server that does not include the BASE_URL are not affected*

**Expected behavior**
The image is displayed correctly, as shown below
![step6](https://user-images.githubusercontent.com/10230144/198281493-19bcb675-4054-404f-b573-2548ebfdad33.png)

**Leantime Version**
2.2.10

**Comment by diderich** [comment]
Note that the issue only occurs if the Leantime server is not installed in the root directory!

**Comment by marcelfolaron** [comment]
Thanks. That was a recent change to allow users to move from one domain to
another. This seems to be a bug.
I will have to rest subfolder installations once more. The thought is that
appURL will need to be set in subfolder installs so my local tests showed
that this case was covered. I will double check

**Comment by marcelfolaron** [comment]
I just did another round of testing and it seems to be working. Here is what I tested. Just in general. The upload procedure removes the main domain from the image and stores the path as `src="download.php..."` 
This should happen in both root folder as well as subfolder installs. 

I have a re-write routine that will then look for src=" that does not start with http and perform an concatination with BASE_URL in the front end layer. (Note: The content has to be displayed using any of the content escape methods from the template class ) Otherwise a call to convertURLs will have to be performed on the string. 

Now with this it is important that appURL is set in the config file if users choose to run Leantime within a subfolder. 

Previously stored images will still contain the absolute path including the domain and will not be updated. It is also still possible to include images from outside sources.
